src/main/java/teammates/logic/api/Logic.java

```diff
@@ -1592,7 +1592,7 @@ public void deleteFeedbackQuestion(String questionId) {
      */
     
     public boolean isQuestionHasResponses(String feedbackQuestionId) {
-        return feedbackQuestionsLogic.isQuestionHasResponses(feedbackQuestionId);
+        return feedbackQuestionsLogic.doesQuestionHaveResponses(feedbackQuestionId);
```

Comment:
You may want to rename the method in this class to be the same as that in `feedbackQuestionsLogic` as well.